x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_actions.tsx

```diff
@@ -27,7 +27,15 @@ export function RuleActions({ ruleActions, actionTypeRegistry }: RuleActionsProp
     ruleActions,
   });
 
-  if (!actionConnectors || actionConnectors.length <= 0)
+  const hasConnectors = useMemo(() => {
```

Comment:
good point, I tend to use `useMemo` quite liberally for anything that is a "computed" property (rather than to memorize them). But I think you're right, these don't have to be in a `useMemo`.